internal/server/config/config_test.go

```diff
@@ -132,6 +132,45 @@ func TestVerifyConfig(t *testing.T) {
 		err := cfg.Verify()
 		require.Error(t, err)
 	})
+
+	t.Run("non_positive_dispatch_throttling_duration", func(t *testing.T) {
+		cfg := DefaultConfig()
+		cfg.DispatchThrottling = DispatchThrottlingConfig{
+			Enabled:             true,
+			TimeTickerFrequency: 0,
+			Rate:                30,
+			Level:               30,
+		}
+
+		err := cfg.Verify()
+		require.Error(t, err)
+	})
+
+	t.Run("non_positive_dispatch_level", func(t *testing.T) {
+		cfg := DefaultConfig()
+		cfg.DispatchThrottling = DispatchThrottlingConfig{
+			Enabled:             true,
+			TimeTickerFrequency: 10 * time.Microsecond,
+			Rate:                30,
+			Level:               0,
+		}
+
+		err := cfg.Verify()
+		require.Error(t, err)
+	})
+
+	t.Run("non_positive_dispatch_level", func(t *testing.T) {
```

Comment:
```suggestion
	t.Run("non_positive_dispatch_rate", func(t *testing.T) {
```

Reply:
obsolete as this test is removed as we no longer have rate